contract/models/contract_line.py

```diff
@@ -559,6 +555,7 @@ def _prepare_invoice_line(self):
             self.last_date_invoiced, self.recurring_next_date
         )
         name = self._insert_markers(dates[0], dates[1])
+
```

Comment:
Don't introduce a useless line break inside a method (and more being unrelated).

Reply:
Removed again. Was left because I could remove the initial code here.